design/style-guide/STYLE-GUIDE.md

```
@@ -497,11 +628,14 @@ Icon sizing within buttons:
   Apply -mx-0.5 to icons for optical alignment
 ```
 
+Broadsheet's large CTA is a distinct tier, not a bigger variant of this table — see "Large CTA + Keycap" below.
+
 ### Badges
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)